.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
remove the rules from eslintrc so they don't drift: https://github.com/jest-community/jest-watch-typeahead/blob/8766546bb3fb26242b49117fb6c9e3f66452b79d/.eslintrc.json#L15-L21